storage-market.md

```diff
@@ -14,7 +14,7 @@ TODO: This is a high level overview of how the storage market interacts with com
 
 ## The Market Interface
 
-This interface, written using Go type notation, defines the set of methods that are callable on the storage market actor. The storage market actor is a built-in network actor. For more information about Actors, see TODO.
+This interface, written using Go type notation, defines the set of methods that are callable on the storage market actor. The storage market actor is a built-in network actor, it can be retrieved using a call to `LoadStorageMarket(state CID)`. For more information about Actors, see [the actors spec](./actors.md).
```

Comment:
saying `LoadStorageMarket(...)` here seems like a bit of an implementation detail to me. If its defined somewhere already, then i'm fine to keep it, otherwise lets just say "it's state can be read from the state tree" or something similar.

Reply:
I get that, we just use it in a lot of pseudocode and it's not clearly defined anywhere. But point well taken.